typedData/tests/example_array.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
where did example_array.json come from?

Reply:
from starknet.js test files
https://github.com/starknet-io/starknet.js/tree/develop/__mocks__/typedData

Reply:
I couldn't find it there, was it removed? Not a big issue though

Reply:
Sorry, you were right!
I take it from the SNIP: https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-12.md#json-example